File: SEBASTIAN_CHAN_00550218/C#/Animales/Vaca.cs
using System.Windows.Forms;
using System;

public class Vaca : Mamifero
{
    private double litrosDeLeche;
    
    public Vaca(string n, double p, double l) : base(n, p)
    {
        this.litrosDeLeche = l;
    }

    public void ImprimirV()
    {
        MessageBox.Show("Nombre de la vaca: " + this.GetNombre() +
        "\nPeso: " + this.peso +
        "\nLitros de leche: "+ this.litrosDeLeche);
    }
}

File: SEBASTIAN_CHAN_00550218/Java/Mamifero/Vaca.java
package Java.ProyectosSuperclass.Mamifero;
import javax.swing.JOptionPane;


public class Vaca extends Mamifero{
    private double litrosDeLeche;
    
    public Vaca(String n, double p, double l){
        super(n,p);
        this.litrosDeLeche=l;
    }

    public void imprimirV(){
        JOptionPane.showMessageDialog(null,"Nombre de la vaca: " + this.getNombre()+
        "\nPeso: " + this.peso +
        "\nLitros de leche: "+this.litrosDeLeche);
    }
    
}
